boards/nuvoton/npcm400_evb/npcm400_evb_defconfig

```diff
@@ -8,6 +8,10 @@ CONFIG_SRAM_VECTOR_TABLE=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=96000000
 CONFIG_XIP=y
 
+# Clock Driver
+CONFIG_CLOCK_CONTROL=y
+CONFIG_CLOCK_CONTROL_NPCM=y
+
```

Comment:
should this be enabled at platform level instead? maybe with a `select CLOCK_CONTROL` in `soc/nuvoton/npcm/npcm4/Kconfig`

`CONFIG_CLOCK_CONTROL_NPCM` should not be needed since it's a default y